frontend/ui/elements/page_turns.lua

```diff
@@ -122,6 +122,15 @@ local PageTurns = {
                     end,
                 })
             end,
+        },
+        {
+            text = _("Invert ui layout mirroring"),
```

Comment:
I'm a bit confused as to what this name is supposed to convey. The widgets will be LTR in RTL documents and vice versa?
```suggestion
            text = _("Invert UI layout mirroring"),
```

Reply:
Might need a less technical/internal naming. And a help_text as when this might be useful, and which widget/features it affects.
No real idea about the name. `Also invert Table of contents` as it's explicite even if incomplete ?

And a enabled_func depending on inverse_reading_order being enabled, so we know one depends on the other.

Reply:
So it means something like `UI follows document writing direction`?

Reply:
Not exactly "writing direction", rather "page turn direction", although it's not the document's (EPUB have a page-turn-direction attribute in the OPF in think, but we don't parse it), it's the user choice of inverse reading order.
The idea is that some users do "inverse reading order", mostly with Arabic and Hebrew books and preferring the English UI for everything. This new option allows them to mirror _some_ UI elements, those showing bits of the content: ToC, Book map, and some related to page turning and reading progress: SkimTo widget.
But for Japanese/Manage, dunno which users will prefer (ie, TOC text might be LTR and better shown in a LTR TOC), so it's an option.

Reply:
I see, so it makes sense combined with the other setting above. The location is good then; I'd suggest adding a separator = true on this one in case something else is added below in the future.

For clarity I'd agree with something like `Also invert table of contents`, `Also invert document-related dialogs` or something along those lines.

Reply:
Thanks for the suggestions,
I updated the text & added a help text, please review :)